### PR TITLE
docs/services/device_mgmt/smp_groups/image_management: fix off

### PR DESCRIPTION
**Pull request by JPHutchins**
The "off" field is not the the last written byte, instead it is the offset of the next byte to be sent.

### CODE REVIEW

**Comment by MaureenHelm** [comment]
@JPHutchins do you plan to return to this PR? please take a look at the comment from @nordicjm

**Comment by JPHutchins** [comment]
> @JPHutchins do you plan to return to this PR? please take a look at the comment from @nordicjm

Thanks for the reminder!

File: doc/services/device_mgmt/smp_groups/smp_group_1.rst [hunk 1]
```diff
@@ -391,7 +391,7 @@ where:
     :align: center
 
     +------------------+-------------------------------------------------------------------------+
-    | "off"            | offset of last successfully written byte of update.                     |
+    | "off"            | Portion of the upload that has been completed, in 8-bit bytes.          |
```
[the remark this answers is not in the document]
Comment (de-nordic) [reply doc/services/device_mgmt/smp_groups/smp_group_1.rst:394]: Would "Offset of the next expected chunk of application image"  be clear enough?

[the remark this answers is not in the document]
Comment (de-nordic) [reply doc/services/device_mgmt/smp_groups/smp_group_1.rst:394]: @JPHutchins Are you still going to make the change here?

File: doc/services/device_mgmt/smp_groups/smp_group_1.rst [hunk 2]
```diff
@@ -295,7 +295,7 @@ where:
     +-----------+--------------------------------------------------------------------------------+
     | "len"     | optional length of an image. Must appear when "off" is 0.                      |
     +-----------+--------------------------------------------------------------------------------+
-    | "off"     | offset of image chunk the request carries.                                     |
+    | "off"     | offset of the next expected chunk of application image.                        |
```
Comment (nordicjm) [inline doc/services/device_mgmt/smp_groups/smp_group_1.rst:298]: I have no idea what's going on here but no

Comment (JPHutchins) [reply doc/services/device_mgmt/smp_groups/smp_group_1.rst:298]: Yeah I was trying to edit from GitHub and found the wrong line or something.  Will fix from an editor ASAP.

**Pull request closed without merging**